__tests__/e2e/simple-e2e.spec.ts

```diff
@@ -8,73 +8,127 @@ describe('Types bundler', () => {
     it('should work with external package dependency', async () => {
         const mockPackages = {
             ...pkg('yury-pkg@1.0.0', `import { Foo } from 'foo-pkg';
-            export type Bla { one: Foo };`, { 'foo-pkg': '1.2.3' }),
+            export type Bla { one: Foo };`, {dependencies: { 'foo-pkg': '1.2.3' }}),
             ...pkg('foo-pkg@1.2.3', `export type Foo { fighters: string };`),
         };
         (fetch as MockedFetch).setMockedNpmPackages(mockPackages);
-        
+
         const result = await bundle('yury-pkg@1.0.0', '/tmp/bundle.d.ts');
 
         expect(result).toBeTruthy();
 
-        expect(removeAllWhiteSpaces(result)).toBe(removeAllWhiteSpaces(`type Foo { fighters: string };
+        expect(removeAllWhiteSpaces(result!)).toBe(removeAllWhiteSpaces(`type Foo { fighters: string };
         type Bla { one: Foo };
-        export { Bla };`));
+        
+        type yurypkg_Bla = Bla;
+        declare namespace yurypkg {
+           export type { yurypkg_Bla as Bla}; 
+        }
+        
+        declare module "yury-pkg" {
+            export = yurypkg;
+        }
+        `));
     });
 
     it('should work with external package dependency with relative .. path import', async () => {
         const mockPackages = {
             ...pkg('yury-pkg@1.0.0', `import { Foo } from 'foo-pkg';
-            export type Bla { one: Foo };`, { 'foo-pkg': '1.2.3' }),
+            export type Bla { one: Foo };`, {dependencies: { 'foo-pkg': '1.2.3' }}),
```

Comment:
`pkg` implementation is:
```ts
export const pkg = (pkgIdentifier: string, typesContents: string, {
  dependencies = {},
  typesName = 'types'
} = {}) => {
...
  const pkgJson = {
    [`${pkgIdentifier}/package.json`]: JSON.stringify({
      name: packageName,
      version: packageVersion,
      [typesName]: types,
      dependencies: {
        ...dependencies
      },
    })
```

so why `dependencies: ...` and not like it was, without the key `dependnecies`?

Reply:
the dependencies are in an object